noaa/challenges/default.json

```diff
@@ -107,7 +107,9 @@
           "warmup-iterations": 100,
           "iterations": 500,
           "target-throughput": 5
-        }
+        }{%- if include_aggs is defined and include_aggs | default(false) %},
```

Comment:
We could be a bit more resilient (and cover both strings and booleans). There are a couple of ways to pass track params in Rally and if you don't pass a json file (or inline json), the KV type definitions could be error prone for boolean type values. In practice, the conditional you have here, evaluates as `True` is there is a a typo like `esrally ... --track-params="include_aggs:FAlse".

Unfortunate standard jinja2 doesn't include the handy `| bool` filter that Ansible contains, but you could instead:

`{% if include_aggs | default(False) | string | lower == "true" %}`

Otherwise, if we want to be string about the datatype being a boolean, Rick's simplification above should work.